CORE-9000 Corda specific values in c5 are pulled out into a separate properties file cordaInternalDependencies.properties with different code owners

E1P3 opened the pull request:
Updating the api version currently requires a codeowners review from a team lead this slows down development quite a bit. this is due to the review rules on gradle.properties.

A new properties file was created [here](https://github.com/corda/corda-runtime-os/blob/jansz/CORE-9000/new-properties-file/cordaInternalDependencies.properties), where we can house c5 specific properties with a different codeowners policy to C5

Change to be applied to corda-runtime-os and [corda-api](https://github.com/corda/corda-api/pull/796) where the new cordaInternalDependecies.properties file can be created, any internal dependencies should be moved to there

E1P3 committed:
adding new properties file

E1P3 committed:
adding new properties file

E1P3 committed:
adding new properties file

ronanbrowne commented on buildSrc/build.gradle:
```diff
@@ -6,10 +6,12 @@ plugins {
 // So we'll load it into our own object to grab what we need.
 def constants = new Properties()
```
```suggestion
Properties constants = new Properties()
```

just for consistency

E1P3 committed:
Update buildSrc/build.gradle

Co-authored-by: Ronan Browne <ronan.browne@R3.com>

E1P3 committed:
Update buildSrc/build.gradle

Co-authored-by: Ronan Browne <ronan.browne@R3.com>

ronanbrowne commented:
@E1P3 can we call this new file `cordaInternal.properties `

E1P3 committed:
Merge branch 'jansz/CORE-9000/new-properties-file' of https://github.com/corda/corda-runtime-os into jansz/CORE-9000/new-properties-file

yift-r3 committed:
CORE-9511: Increase timeout (#2935)

* CORE-9511: Increase timeout

* Increase timeout

kyriathar-r3 committed:
CORE-7309 Api to find my signing keys (#2912)

* Implements find my signing keys API on `SigningServiceImpl`. Delegates to existing underlying infrastructure. Finds owned keys and then creates a mapping from requested key to found to be owned key or null if key is not owned by caller. In case of composite keys it maps the composite key to the first found key leaf.

* Converts requested keys to their IDs to create Kafka request containing the key IDs instead serialized keys.

kyriathar-r3 committed:
Remove duplicate test reference to same service (#2942)

E1P3 committed:
NOTICK:Updated api version

E1P3 committed:
NOTICK:Updated api version

E1P3 committed:
Merge branch 'release/os/5.0' into jansz/CORE-9000/new-properties-file

vlajos committed:
NOTICK Ledger test cleanups (#2931)

* Remove a few duplicated dependencies
* Use slightly more compact `wireTransactionFactory.createExample` in ledger integration tests.
* Reuse CPK example and signature example classes in a few more places.
* CORE-7649 Remove a few package-info.javas

pmck91 committed:
CORE-8017 - Rename `RPCStartableFlow` and `RPCRequestData` (#2909)

* Refactoring RPCStartableFlow to RestStartableFlow and RPCRequestData to RestRequestBody

* RestStartableFlow -> ClientStartableFlow

* missed RestStartedFlow -> ClientStartedFlow.kt

* fixing deserializer

* fixing ClusterBuilder and RPC RequestDataFactoryWrapperTest

* Update simulator/runtime/src/main/kotlin/net/corda/simulator/runtime/serialization/SimpleJsonMarshallingService.kt

Co-authored-by: Viktor Kolomeyko <viktor.kolomeyko@r3.com>

* incorrect parameter in start flow params

* updating apibaseline

* missing request body updated

* updated api version

Co-authored-by: Viktor Kolomeyko <viktor.kolomeyko@r3.com>

chrisr3 committed:
NOTICK: Allow Jenkins to configure PostgreSQL using Gradle properties. (#2941)

josephzunigadaly committed:
CORE-8336: Add description for packaging sign command (#2605)

- Add description for sign command
- Capitalise the input file parameter description

emilybowe committed:
CORE-9124: Flow tries to use fiber before it is initialised (#2938)

A fiber was being referenced before it was initialised within the exception handler. This PR checks if the flow exists (using doesExist) and if false only returns the flow ID. If the flow exists the flow start context is returned also (potentially more useful).

yift-r3 committed:
CORE-8874: Add MGM client certificate subject to the saved group policy file (#2927)

* CORE-8874: Add MGM client certificate subject to the saved group policy file

* Apply review comment

* Apply review comments, fix failing deployment

* Fix detekt issue

* Bump API version

nkovacsx committed:
Use publicKeyId instead of Base58 (#2943)

E1P3 committed:
file rename

E1P3 committed:
file rename

E1P3 committed:
Merge branch 'jansz/CORE-9000/new-properties-file' of https://github.com/corda/corda-runtime-os into jansz/CORE-9000/new-properties-file

E1P3 committed:
Merge branch 'jansz/CORE-9000/new-properties-file' of https://github.com/corda/corda-runtime-os into jansz/CORE-9000/new-properties-file

chrisr3 committed:
NOTICK: Fix technical debt. (#2948)

E1P3 committed:
Merge branch 'release/os/5.0' into jansz/CORE-9000/new-properties-file

E1P3 committed:
Merge branch 'jansz/CORE-9000/new-properties-file' of https://github.com/corda/corda-runtime-os into jansz/CORE-9000/new-properties-file

E1P3 committed:
Merge branch 'jansz/CORE-9000/new-properties-file' of https://github.com/corda/corda-runtime-os into jansz/CORE-9000/new-properties-file
